src/TypeDefinition.js

```diff
@@ -381,6 +381,7 @@ export type NavigationTransitionProps = {
   // is the index of the scene
   scene: NavigationScene,
   index: number,
+  navigation: NavigationScreenProp<NavigationRoute, NavigationAction>,
```

Comment:
All righty, sounds good! Thanks for clarifications, some of the things are still mythical to me!